distribution/lib/Standard/Base/0.2.32-SNAPSHOT/src/Data/Vector.enso

```diff
@@ -603,6 +601,26 @@ type Vector
             if this.length == 1 then prefix + this.unsafe_at 0 + suffix else
                 prefix + this.unsafe_at 0 + (1.up_to this.length . fold "" acc-> i-> acc + separator + this.unsafe_at i) + suffix
 
+    ## Creates a new vector with the skipping elements until `start` and then
+       continuing for `length` elements.
+
+       Arguments:
+       - start: The number of elements to drop from the start of `this`.
+       - length: The number of elements for the sub-vector.
+
+       > Example
+         Remove the first 2 elements then take 5 from the vector.
+
+             [1, 2, 3, 4, 5, 6, 7, 8].slice 2 5
+    slice : Integer -> Integer -> Vector Any
+    slice start length =
```

Comment:
I think everywhere I've seen a `slice` operation (currently I remember Python's `arr[a:b]` and JS's `arr.slice(a,b)`, the second argument is the `end` position and not the length to copy. I.e. `0.up_to 10 . slice 2 5 == [2, 3, 4]` in these languages. Don't we want to be consistent with that too?

I think that a variant accepting the first position + length may also be worth having - but I'd consider giving it a different name to avoid confusion for users coming from other languages.

Reply:
renamed to `take` and now is start end in keeping with the proposal for Text.
Will be revisited as part of normalising the APIs between Text and Vector